config/pipeline.yaml

```
@@ -622,6 +622,96 @@ jobs:
       job_timeout: 10
     kcidb_test_suite: kselftest.iommu
 
+  preempt-rt: &preempt-rt
+    template: preempt-rt.jinja2
+    kind: test
+    # TODO: Update NFS root to correct one (debian_bookworm-rt_ramdisk)
+    nfsroot: 'http://storage.kernelci.org/images/rootfs/debian/bookworm-kselftest/20240313.0/{debarch}'
+    filters:
+      - passlist: {defconfig: ['preempt_rt']}
+    params: &preempt-rt-params
+      job_timeout: '10'
+      duration: '60s'
```

This would fix the validation error.

```suggestion
  preempt-rt: &preempt-rt
    template: preempt-rt.jinja2
    kind: test
    # TODO: Update NFS root to correct one (debian_bookworm-rt_ramdisk)
    nfsroot: 'http://storage.kernelci.org/images/rootfs/debian/bookworm-kselftest/20240313.0/{debarch}'
    filters:
      - passlist: {defconfig: ['preempt_rt']}
    params: &preempt-rt-params
      job_timeout: '10'
      duration: '60s'
   kcidb_test_suite: preempt-rt
```